### PR TITLE
EPMRPP-35338 add launch number to some widgets name

### PR DESCRIPTION
**Pull request by twinfreaks**

### CODE REVIEW

File: app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
-);
+const localMessages = defineMessages({
+  launchNameText: {
+    id: 'LaunchExecutionAndIssueStatistics.launchNameText',
+    defaultMessage: 'Launch name:',
+  },
+});
+
+export const LaunchExecutionAndIssueStatistics = injectIntl((props) => {
+  const { result = {} } = props.widget.content;
```
Comment (DiscoElevator) [inline app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx]: Maybe `result = []` (according to the line below)?

File: app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.scss [hunk 2]
```diff
@@ -44,6 +43,37 @@
...
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+  margin-top: -20px;
+}
+
+.launch-name-block {
+  display: inline-flex;
+  height: 40px;
+  z-index: 1;
```
Comment (DiscoElevator) [inline app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.scss]: $Z-INDEX-DEFAULT

File: app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx [hunk 3]
```diff
@@ -1,17 +1,40 @@
...
+  const { name, number } = result[0];
+  const launchName = number ? `${name} #${number}` : name;
```
Comment (DiscoElevator) [inline app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx]: Doesn't work for case when `number` is 0

File: app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx [hunk 4]
```diff
@@ -223,10 +223,13 @@ export class PassingRatePerLaunch extends Component {
...
+    const { latestLaunch: { number: launchNumber } = {} } = widget.content;
 
-    return this.isRatePerLaunchType()
-      ? widget.contentParameters.widgetOptions.launchNameFilter
-      : widget.appliedFilters && widget.appliedFilters[0].name;
+    if (this.isRatePerLaunchType()) return widget.contentParameters.widgetOptions.launchNameFilter;
+
+    const launchName = widget.appliedFilters && widget.appliedFilters[0].name;
+
+    return launchNumber ? `${launchName} #${launchNumber}` : launchName;
```
Comment (DiscoElevator) [inline app/src/components/widgets/charts/passingRatePerLaunch/passingRatePerLaunch.jsx]: Doesn't work for case when `launchNumber` is 0

File: app/src/components/widgets/mostTimeConsumingTestCases/mostTimeConsumingTestCases.jsx [hunk 5]
```diff
@@ -79,6 +82,8 @@ export class MostTimeConsumingTestCases extends Component {
       container,
     } = this.props;
 
+    const launchName = number ? `${name} #${number}` : name;
```
Comment (DiscoElevator) [inline app/src/components/widgets/mostTimeConsumingTestCases/mostTimeConsumingTestCases.jsx]: Doesn't work for case when `number` is 0

File: app/src/components/widgets/tables/components/testsTableWidget/testsTableWidget.jsx [hunk 6]
```diff
@@ -8,14 +8,18 @@ import styles from './testsTableWidget.scss';
 
 const cx = classNames.bind(styles);
 
-export const TestsTableWidget = ({ launch, tests, issueType, columns, hideInfoBlock }) => (
-  <div className={cx('tests-table-widget')}>
-    <div className={cx('widget-wrapper')}>
-      {!hideInfoBlock && <LaunchInfoBlock launchName={launch.name} issueType={issueType} />}
-      <TestsTable columns={columns} tests={tests} launchId={launch.id} />
+export const TestsTableWidget = ({ launch, tests, issueType, columns, hideInfoBlock }) => {
+  const launchName = launch.number ? `${launch.name} #${launch.number}` : launch.name;
```
Comment (DiscoElevator) [inline app/src/components/widgets/tables/components/testsTableWidget/testsTableWidget.jsx]: Doesn't work for case when `number` is 0

File: app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx (hunk 3 above)
Comment (twinfreaks) [reply app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx]: Could case number be 0?

Comment (DiscoElevator) [reply app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatistics.jsx]: Clarified with BA - launch number cannot be 0

**Pull request merged**